schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "FOCAL2PIXEL_Y": {
+           "description": "Transformation y coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "SENSOR_LOCATION": {
+           "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+      },
+       "SENSOR_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                    "type": "number",
+                    "minItems": 3,
+                    "maxItems": 3
+		                }
+	                 }
+	     },
+       "SUN_POSITION" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+	     },
+       "SUN_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		        }
+	         }
+	     },
+
+       "ORIENTATION" : {
+           "description": "Orientation of the sensor provided in quaternions",
+           "type": "array",
+           "items": {
+               "type": "array",
+               "items" : {
+                  "type": "number",
+                  "minItems": 4 ,
+                  "maxItems": 4
+           }
+          }
+      },
+
+       "DETECTOR_LINE_RATE": {
+           "description": "The line rates of the detector map with the associated start line and time",
+           "type": "array",
+              "items": {
+	               "type": "object",
+	                "properties": {
+	                   "START_LINE": {
+		                     "type": "number"
+	                    },
+                      "LINE_START_TIME": {
+ 	                        "type": "number"
+	                     },
+	                     "EXPOSURE_TIME": {
+		                       "type": "number"
+	                     },
+	          "required": [ "EXPOSURE_TIME"]
+	                 }
+	              }
+      },
```

Comment:
This table could have thousands of entries in it. I'm concerned about having to have `{START_LINE: value, LINE_START_TIME: value, EXPOSURE_TIME:}` duplicated for each entry. Would it be reasonable to convert this to a 2d array like the ephemeri?

Reply:
Yes, I will do that.